custom_components/alexa_media/strings.json

```diff
@@ -102,5 +102,22 @@
         }
       }
     }
+  },
+  "issues": {
+    "restart_required": {
```

Comment:
Where is this used?

Reply:
I copied code from HACS when they deprecated YAML but I realize now that my other edits don't use it (now that you've pointed it out) and have removed that section.